### PR TITLE
Mention support in Helix editor

### PR DESCRIPTION
**Pull request by pimeys**
Support added last summer: https://github.com/helix-editor/helix/pull/2703

### CODE REVIEW

File: content/300-guides/125-development-environment/100-editor-setup.mdx [hunk 1]
```diff
@@ -46,3 +46,7 @@ You can install the official [Prisma VS Code extension](https://marketplace.visu
 ### nova
 
 - [nova](https://extensions.panic.com/extensions/robb-j/robb-j.Prisma/) provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server.
+
+### Helix
+
+- [Helix](https://helix-editor.com/) from version 22.08 provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language server.
```
Comment (pimeys) [inline content/300-guides/125-development-environment/100-editor-setup.mdx]:
```suggestion
- [Helix](https://helix-editor.com/) from version 22.08 provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server.
```

**Review by janpio — APPROVED** [review]

**Comment by janpio** [comment]
Why the change @zachtil? All previous entries list the editor in question first. The version is super unimportant information.

**Comment by zachtil** [comment]
@janpio Ok, I can change that. My consideration was related to the fact that the sentence structure feels somewhat difficult to read. If I had to say it out loud, I'd probably pause before and after 'from version 22.08'. That typically implies you need surrounding commas. So my feeling was that it breaks the flow of the sentence a bit.

Thinking about it a bit more, perhaps we have these options. I went for **a** because I thought that we wanted to indicate (somewhat early in the sentence) the version from which the Prisma-supportive features are available.

a. _From version 22.08, Helix provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server._
b. _Helix, from version 22.08, provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server._
c. _Helix (from version 22.08) provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server._
d. Helix provides syntax highlighting of the Prisma Schema Language and uses the Prisma Language Server from version 22.08.

**Comment by janpio** [comment]
I generally think that unless there is an assumption most people are on other versions than the newest the version info should be more an aside, so I would go with c or d here.

**Comment by zachtil** [comment]
Thanks, Jan. I will go with c as I am just realizing that at the end of the sentence, there might be confusion as to whether the version info applies to Helix, PSL, or PLS.

**Pull request merged**